matter_server/client/matter.py

```diff
@@ -1,250 +0,0 @@
-"""Matter base class."""
```

Comment:
Discussed offline with Marcel. We're going to re-instate this file again to split again the Matter from WebSocket logic.